server/src/main/java/io/crate/protocols/postgres/types/PGTypes.java

```diff
@@ -56,6 +56,7 @@
         .put(DataTypes.TIMETZ, TimeTZType.INSTANCE)
         .put(DataTypes.TIMESTAMPZ, TimestampZType.INSTANCE)
         .put(DataTypes.TIMESTAMP, TimestampType.INSTANCE)
+        .put(DataTypes.DATE, TimestampType.INSTANCE)
```

Comment:
To support the `DATE` type on the postgres protocol, a dedicate PgType must be implemented using the same OID postgres is using (`1082`).

Reply:
Done